src/pilot/deploy-overcloud.py

```diff
@@ -637,6 +278,10 @@ def main():
                             dest="hugepages_size",
```

Comment:
There might be a check here. What if the user enable hugepages without specifying any values for hugepage_size? What is expected to happen then?

Reply:
Please refer my above comment

Reply:
Fixed
Added default values.